app/controllers/conference_controller.rb

```diff
@@ -1,6 +1,13 @@
 class ConferenceController < ApplicationController
   before_filter :respond_to_options
   load_and_authorize_resource find_by: :short_title
+  skip_before_filter :verify_authenticity_token, if: :json_request?
```

Comment:
Why is this needed? verify_authenticity_token is for verification of the CSRF token for posting things, which you don't do.